guides/process/scheduling/setting_up_projects.md

```diff
@@ -0,0 +1,127 @@
+
+- GRP - group (typically Service Areas)
+- NOL - no location (e.g. paid time off, absence, Learn Tech)
+- LDN - London
+- MCR - Manchester
+- BRS - Bristol
+
+### Category
+
+A three letter code identifying the the category of the project. Billed, Bench and Investment are described in more detail below.
+
+- BIL - Billed to client
+- BEN - Bench (AKA chalet) 
+- INV - Investment
+- INT - Internal (typically a Service Area)
+- SAM - Sales and Marketing
+- PTO - Paid time off
+
+### Subcategory
+
+An optional three letter code identifying the subcategory. These are currently used to distinguish different types of internal project:
+
+- ACA - Academy
+- REC - Recruitment
+- WEL - Culture and Happiness
+- LAD - Learning and Development
+
+### Number
+
+This is a number that makes the project code unique. It's important to ensure your project code has not been used before. Harvest will warn you of this, but will not prevent you from creating a project with a duplicate code.
+
+The number confers no meaning.
+
+### Examples
+
+- LDN-BIL-01 - a billed project in London
+- MCR-BEN-01 - a bench/chalet project in Manchester
+- BRS-INV-01 - an investment project in Bristol
+
+
+### Billed, Bench, and Investment
+
+Billed, Bench, and Investment are the primary project categories that we set up day-to-day.
+
+There are some important distinctions:
+
+#### Billed
+
+This is a typical project that is billed back to the customer. Any time that is allocated to this project will end up on an invoice at the end of the month. There are a number of must-haves for a billed project:
+
+- Everybody assigned to the project must have a defined day rate
+- Nobody assigned to a billed project must have a zero rate (use an Investment or Bench project for this instead)
+- The project must have a budget defined
+
+#### Bench
+
+This is a project to make better use of underutilisation in the team. This could be for team members who are unbilled on customer delivery but wouldn't qualify as an investment, or for an internal initiative.
+
+Team members who are assigned bench projects will negatively reflect on a market's utilisation rate. If a team member from one market works on a bench project in another market they will not be reimbursed for their time.
+
+#### Investment
+
+This is a project that captures proactive investment in an account or workstream. An investment must be a conscious and proveable decision for a longer-term benefit. A team member simply being underutilised should be allocated to a bench project.
+
+Team members who are assigned investment projects will not negatively reflect on a market's utilisation rate. If a team member from one market works on an investment project in another market they will be reimbursed for their time.
+
+
+## Naming convention
+
+Primarily projects should be named to make it easy at a glance to understand what it is. If it covers a particular phase, or a budget extension, please encode this in the project name.
```

Comment:
Is this the place to add extensions to projects should be treated as a separate projects where there is a different budget?